pkg/networkmanager/firewall.jsx

```diff
@@ -932,57 +841,47 @@ export class Firewall extends React.Component {
         });
         services.sort((a, b) => a.name.localeCompare(b.name));
 
-        var zones = [...this.state.firewall.activeZones].map(id => {
+        var zones = [...this.state.firewall.activeZones].sort((z1, z2) =>
+            z1 === firewall.defaultZone ? -1 : z2 === firewall.defaultZone ? 1 : 0
+        ).map(id => {
```

Comment:
oh, so point of this ternary operator inside ternary operator is to make the default zone first in the list? (I don't have better idea how to do it, but maybe comment on top would not hurt :) )
Also is it somehow interesting for user that the zone is default? Because from UI it is not visible that the zone is default.

Reply:
On home machines or machines with a single interface it's mostly likely the zone you want, so that's why it's on top. There's not really anything special about it other than it being probably the one you want when the network setup is simple.